container/cli.py

```diff
@@ -264,14 +260,19 @@ def __call__(self):
             LOGGING['loggers']['container']['level'] = 'DEBUG'
         config.dictConfig(LOGGING)
 
+        vargs = vars(args)
+        if not vargs['project_name']:
```

Comment:
No idea what you mean. The point is to set the `project_name` value in a single place. Prior to this we set it 18 times in `core.py`.